config/community_images.json

```diff
@@ -17,4 +17,14 @@
     "image": "us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:3.13.2",
     "requiresSpark": false,
     "isRStudio": true
+},
+{
+    "id": "OpenVINO",
+    "label": "OpenVINO (OpenVINO 0.5.0, Python 3.7.10, GATK 4.2.0.0)",
+    "version": "0.1.0",
+    "updated": "2021-08-31",
+    "packages": "https://storage.googleapis.com/terra-docker-image-documentation/terra-jupyter-gatk-ovtf-0.1.0-versions.json",
```

Comment:
This file contains all the Python/R installed packages and their versions. (it includes `openvino_tensorflow`)

Reply:
(The packages will be displayed when you click `What’s installed on this environment?` in the UI)